athenaCL/libATH/athenaObj.py

```diff
@@ -15,9 +15,9 @@
 import sys, os, time, random, traceback, http.client, urllib.request, urllib.parse, urllib.error
 import unittest, doctest
 
-athVersion = '2.0.0a15'
-athBuild = '2010.07.07'
-athDate = '7 July 2010' # human readable version
+athVersion = '2.0.0a16'
+athBuild = '2021.04'
```

Comment:
Sorry, but again it violates the current versioning system... While it has 'YEAR.MONTH.DAY' format, yours just 'YEAR.MONTH'. Check `athDate` as well. In overall, I don't think we should've updated the version number as there are no plans for a new release now, but ok — we'll continue working in the scope of the new version number.